lib/command_handler.rb

```diff
@@ -0,0 +1,20 @@
+require 'faraday'
+require 'json'
+require './lib/help_command_handler'
+
+class CommandHandler
+    Faraday.default_adapter = :net_http
+
+    def initialize(arguments_string)
+        @arguments_string = arguments_string
+    end
+
+    def handle_command
+        HelpCommandHandler.new.print_help_message
+    end
+
+    def get_from_url(url)
```

Comment:
I'd prefer using Composition over Inheritance to share functionality.

To be able to reuse this method that does HTTP GET calls in another class, you'd need to inherit that class from CommandHandler.
Another approach is to move the `get_from_url` class to a separate class that does HTTP calls and use that.

There may be different layers of abstraction that one can apply:
HTTPClient < exposes methods like GET / POST / etc..
RubyGemsClient < exposes methods like `search` or `info` which hide the fact from the caller the implementation details that it uses HTTP in the background. 

RubyGemsClient is a higher level abstraction than HTTPClient that allows the user of the class to focus on functionality without caring about which endpoint it needs to call.